lib/ProductOpener/Tags.pm

```diff
@@ -1187,6 +1192,10 @@ sub put_file_to_cache ($source, $target) {
 		if (!$response->is_success()) {
 			print "Error uploading to GitHub cache for $target: ${\$response->message()}\n";
 		}
+		elsif ($target =~ /\.result\.sto$/) {
+			# Only give one message rather than one for each individual file
+			print "Uploaded $target to GitHub cache\n";
```

Comment:
```suggestion
			print "Uploaded $target.gz to GitHub cache\n";
```

Reply:
Keeping as is for symmetry with the "Fetched" log and to avoid further commits. Also I don't list all files so it's really just so we can see the generated hash